cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/seeds/SeedsManager.java

```diff
@@ -159,8 +160,16 @@ public SeedsManager(final DefaultConfigurationManager configurationManager,
         final List<String> seeds = new ArrayList<>(active.size());
 
         for (int seed = 0; seed < seedCount && seed < active.size(); ++seed) {
-            seeds.add(InetAddress.getByName(active.get(seed).getHostname())
-                    .getHostAddress());
+            CassandraDaemonTask daemon = active.get(seed);
+            Protos.ContainerStatus status = daemon.getTaskStatus().getContainerStatus();
+            if (status.getNetworkInfosCount() == 0) {
+                seeds.add(InetAddress.getByName(daemon.getHostname())
+                        .getHostAddress());
+            } else {
+                String ip = status.getNetworkInfos(0).getIpAddresses(0).getIpAddress();
```

Comment:
I think joining only one CNI network at a time makes sense.  I don't the current configuration options exposed to users would allow more than 1 correct?